builders/interact/eth-libraries/deploy-contract.md

```diff
@@ -1,44 +1,47 @@
 ---
 title: 部署合约
-description: 学习如何使用Web3.js、Ethers.js或Web3.py，通过简单脚本将未经改动的基于Solidity的智能合约部署到Moonbeam节点。
+description: 通过此教程学习如何使用Web3.js、Ethers.js或Web3.py的简单脚本将未经改动的基于Solidity的智能合约部署到Moonbeam节点。
```

Comment:
There are 5 tables and a code snippet that aren't rendering correctly on this page

Reply:
see the comment above for send-transaction.md